scanomatic/data_processing/subplates.py

```diff
@@ -1,8 +1,12 @@
+from typing import Optional, Union
 import numpy as np
 
 from . import mock_numpy_interface
 
 
+TUPLE_TYPE = Union[tuple[int, int], tuple[int, int, int]]
```

Comment:
maybe `PlateShape` or `Plate` is better type name?

Reply:
agree. changed to PlateShape.